### ISSUE TITLE
Direction does not change with `direction` attribute

### ISSUE THREAD

**Issue by scottnath** [colinaut/action-table#1]
Hi @colinaut,
Thanks for a deeply thought out component.

I believe there is a bug where initially adding an attribute to `<action-table direction="descending">` does not change the direction from ascending.

The `this.direction = "ascending";` [line in the action-table constructor](https://github.com/colinaut/action-table/blob/main/src/action-table.ts#L15) negates the need for the `get direction()` method to fire. I can change direction by adding a urlparam and by localstorage, but the attribute did not work when attempting to set it as `descending` initially.

The workaround I did was to change the attribute _after_ the rest of the DOM is changed:

```
<script>
	const changeDir = () => {
		const table = document.querySelector('action-table');
		table.setAttribute('direction', 'descending');
	}
	async function getBtn() {
		return new Promise(resolve => {
			const sir = setInterval(() => {
				const table = document.querySelector('button[data-col="column-name"]');
				if (table != '') {
					clearInterval(sir);
					resolve(changeDir());
				}
			}, 100);
		});
	}
	getBtn();
</script>
```

thanks,
scott

**Comment by colinaut** [comment]
Ok thanks I'll check on that. Likely a bug introduced in my recent tweaks to how the table handles initialization. Should be easily fixable

**Comment by colinaut** [comment]
Fixed in 2.3.9! I also discovered a bug that made aria-sort not switch. That's been fixed too. https://github.com/colinaut/action-table/releases/tag/2.3.9

**Comment by scottnath** [comment]
Worked fantastic - thank you @colinaut!
